### PR TITLE
add hacktoberfest wrapup blog post

### PR DESCRIPTION
**Pull request by amberleyromo**
hacktoberfest wrapup blog post

### CODE REVIEW

File: docs/blog/2018-11-01-hacktoberfest-wrapup/index.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+To split our time more efficiently, we [created codeowner teams](with-great-contributions-comes-great-responsibility) to automatically notify Inkteam members when certain areas of review were required:
+
+- Core - Code owners for core functionality of Gatsby.
+- Docs - Maintainers who have special expertise in docs.
+- Ecosystem - Code owners for the Gatsby ecosystem.
+- Website - Maintainers who keep gatsbyjs.org healthy.
+
+#### ➡️ We're hiring another Open Source Maintainer!
+
+We knew this was an area of need that will continue to grow (which is great!) This role will focus on helping us improve Gatsby core, the Gatsby ecosystem and support the community 🎉 Check out the [full job spec](https://www.gatsbyjs.com/careers/open-source-maintainer/) for more details.
```
Comment (DSchau) [inline docs/blog/2018-11-01-hacktoberfest-wrapup/index.md]:
```suggestion
We knew this was an area of need that will continue to grow (which is great!). This role will focus on helping us improve Gatsby core, the Gatsby ecosystem, and support the community 🎉 Check out the [full job spec](https://www.gatsbyjs.com/careers/open-source-maintainer/) for more details.
```

File: docs/blog/2018-11-01-hacktoberfest-wrapup/index.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+#### ➡️ Create more accessible Hacktoberfest issues
+
+Next year, we could focus on curating a backlog of accessible Hacktoberfest issues that get released in waves through the month.
+
+#### ➡️ Start planning earlier!
+
+This year was a bit of an experiment. We have a lot of great lessons learned to put into play next year!
+
+## Wrapup
```
Comment (DSchau) [inline docs/blog/2018-11-01-hacktoberfest-wrapup/index.md]:
```suggestion
## Wrap-up
```

nit-pick 🙃

Comment (DSchau) [reply docs/blog/2018-11-01-hacktoberfest-wrapup/index.md]: Or Wrap up 🤔

File: docs/blog/2018-11-01-hacktoberfest-wrapup/index.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+## Things to learn from
+
+### We were overwhelmed
+
+We’re a small team, and we weren’t fully prepared to handle the overwhelmingly enthusiastic response from the community. Many issues, particularly docs issues, were claimed within minutes of opening them. It slowed our response time, which was frustrating -- both for contributors, and for us.
+
+#### ➡️ We created codeowner teams
+
+To split our time more efficiently, we [created codeowner teams](with-great-contributions-comes-great-responsibility) to automatically notify Inkteam members when certain areas of review were required:
```
Comment (DSchau) [inline docs/blog/2018-11-01-hacktoberfest-wrapup/index.md]: Are people supposed to know what Inkteam means? Maybe just rephrase as team members or something?

File: docs/blog/2018-11-01-hacktoberfest-wrapup/index.md (hunk 1 above)
Comment (DSchau) [reply docs/blog/2018-11-01-hacktoberfest-wrapup/index.md]: Oxford comma for life

File: docs/blog/2018-11-01-hacktoberfest-wrapup/index.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ author: "Amberley Romo"
 tags: ["hacktoberfest", "open source"]
 ---
 
-Just like that, 👻🎃 Gatsby Hacktoberfest 🎃👻 is over! Or, "Hacktoverfest"? 😂 (Thanks for the pun, [@m-allanson](https://github.com/m-allanson)). We had a great time Hacktoberfesting with y'all and wanted to share a bit of our planning process and lessons learned in review!
```
Comment (DSchau) [inline docs/blog/2018-11-01-hacktoberfest-wrapup/index.md]: I kinda liked the emoji 😢

File: docs/blog/2018-11-01-hacktoberfest-wrapup/index.md (hunk 1 above)
Comment (amberleyromo) [reply docs/blog/2018-11-01-hacktoberfest-wrapup/index.md]: you oxford comma people! (fixed, lol).

**Pull request merged**